dev/io.openliberty.faces.fat.selenium.util.internal/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
```

Comment:
Why was JavaSE-11 chosen?

Reply:
Just copied from here: https://github.com/OpenLiberty/open-liberty/blob/45a3ca07f01c6139278ed1a0721539eb9c463fbc/dev/io.openliberty.jakarta.servlet.6.0/.classpath

Reply:
My bad. We should have copied it from a project which uses Java 8.